README.md

```diff
@@ -43,6 +44,9 @@ In addition, the following items are optional for each user:
 * shell - The user's shell. This defaults to /bin/bash. The default is
   configurable using the users_default_shell variable if you want to give all
   users the same shell, but it is different than /bin/bash.
+* main_group - If provided, this will be the primary group for the user.
+* force_passwd_update - Boolean value, if true, this will cause the password
```

Comment:
I think it's a little clearer if you just call this `update_password` and wrapper the modules functionality (takes either the string 'always' or 'on_create' as a value.